lego-webapp/pages/lending/+Page.tsx

```
@@ -33,22 +35,43 @@ const LendableObject = ({
+                {lendableObject.location}
+              </p>
+            </div>
+          </div>
         </div>
       </Card>
     </a>
   );
 };
 
+const truncateText = (text: string, maxLength: number): string => {
+  return text.length > maxLength ? text.slice(0, maxLength) + '...' : text;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)